proto/internal/temporal/server/api/taskqueue/v1/message.proto

```diff
@@ -77,7 +79,7 @@ message PhysicalTaskQueueInfo {
     // Unversioned workers (with `useVersioning=false`) are reported in unversioned result even if they set a Build ID.
     repeated temporal.api.taskqueue.v1.PollerInfo pollers = 1;
     temporal.api.taskqueue.v1.TaskQueueStats task_queue_stats = 2;
-    InternalTaskQueueStatus internal_task_queue_status = 3;
+    repeated InternalTaskQueueStatus internal_task_queue_status = 3;
```

Comment:
Similarly, doesn't this break for existing callers?

Reply:
Oh, interesting. I didn't know that about protos.

Reply:
It looks like buf breaking complains about it, but it's too picky, we can ignore it